### PR TITLE
Add docker labels

### PR DESCRIPTION
**Pull request by ptqa**
Adds var to add docker labels

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -155,6 +155,13 @@ variable "container_depends_on" {
   default     = []
 }
 
+variable "docker_labels" {
+  type        = "map"
+  description = "The configuration options to send to the `docker_labels`"
+
```
Comment (aknysh) [inline variables.tf]:
```suggestion
```

File: outputs.tf [hunk 2]
```diff
@@ -12,15 +12,17 @@ locals {
   encoded_memory_reservation    = "${var.container_memory_reservation > 0 ? var.container_memory_reservation : "null"}"
   encoded_stop_timeout          = "${var.stop_timeout > 0 ? var.stop_timeout : "null"}"
   encoded_container_definition  = "${replace(replace(replace(jsonencode(local.container_definition), "/(\\[\\]|\\[\"\"\\]|\"\"|{})/", "null"), "/\"(true|false)\"/", "$1"), "/\"(-?[0-9]+\\.?[0-9]*)\"/", "$1")}"
+  encoded_docker_labels         = "${jsonencode(var.docker_labels)}"
```
Comment (aknysh) [inline outputs.tf]: @ptqa thanks
will it work correctly with an empty map `{}` (the default value)?
should it be encoded as in the comment above #  - Convert `""`, `{}`, `[]`, and `[""]` to `null`
and as done for `encoded_container_definition`?

Comment (ptqa) [reply outputs.tf]: @aknysh I've just tested again. It works fine if I don't specify this parameter and use default. In AWS you get task registered with ` "dockerLabels": {},`.

Not sure what this replace magic does for `encoded_container_definition`, can you explain why we need it here? What is the use case?

**Comment by jamengual** [comment]
Well this is exactly what I was asking in the Slack channel, is there something I can help with ? testing ?

**Comment by ptqa** [comment]
@aknysh ?

**Comment by statwoodland** [comment]
Anything blocking this? I'm using this module quite extensively and need to add dockerLabels.

**Pull request merged**